generator/api/src/main/java/io/fabric8/maven/generator/api/support/BaseGenerator.java

```diff
@@ -114,7 +107,7 @@ protected String getConfig(Configs.Key key, String defaultVal) {
 
     // Get 'from' as configured without any default and image stream tag handling
     protected String getFromAsConfigured() {
-        return getConfigWithFallback(Config.from, FABRIC8_GENERATOR_FROM, null);
+        return getConfigWithFallback(Config.from, "fabric8.generator.from", null);
```

Comment:
Why did you replace constants with magic strings? This will be reported as a code smell by any static code analysis tool.

Reply:
Oh I did this mistake while rebasing with master. Removed the constants to see what happens, then I think IDE replaced that with hard-coded strings instead of reporting errors.